omnigibson/robots/r1.py

```diff
@@ -237,9 +237,24 @@ def finger_joint_names(self):
     def usd_path(self):
         return os.path.join(gm.ASSET_PATH, "models/r1/r1.usd")
 
+    @property
+    def curobo_path(self):
+        return {
```

Comment:
Can you update the original docstring in `curobo_path`? Currently it expects this to be a string, not a dict

Reply:
also maybe rename to `curobo_paths`?